packages/lit-dev-tests/package.json

```diff
@@ -32,6 +32,33 @@
       ],
       "command": "../../node_modules/.bin/tsc --build --pretty",
       "clean": "if-file-deleted"
+    },
+    "start": {
+      "dependencies": [
+        "../..:start"
+      ]
+    },
+    "test:integration": {
+      "dependencies": [
+        "build"
+      ],
+      "files": [
+        "src/**/*.png"
+      ],
+      "command": "playwright install && playwright test --config=lib/playwright.config.js"
+    },
+    "test:integration:update-golden-screenshots": {
+      "dependencies": [
+        "build"
+      ],
+      "files": [],
```

Comment:
Add `"output": []` so that it can be cached if nothing changed?

Reply:
This updates the screenshots, so it needs that as output, done